r-package/paper2020/paper2020.Rmd

```diff
@@ -144,32 +154,68 @@ ggplot(data=plot_data, aes(y = travel_time, x = reorder(fromId, mediantt))) +
 
 #### Visualizing Isochrones
 
-In our example, we can visualize the isochrone departing from the central bus station as follows:
+In our example, we can visualize the isochrone (area reachable within a certain amount of time) departing from the central bus station as follows:
+
+<!-- Travel time percentile 50 is the median, we shouldn't be calculating it from the other travel times -MWC -->
+<!-- For this map and the next, I couldn't figure it out but there must be a way to crop the map view so only the shaded
```

Comment:
Good point. @mvpsaraiva , do you know a simple way to do this?

Reply:
done in f259d48